lib/api/core/plugins/pluginsManager.js

```diff
@@ -967,7 +963,7 @@ class PluginsManager {
    * @param {object} plugin
    * @throws {PluginImplementationError} If strategies registration fails
    */
-  _initAuthenticators (plugin) {
+  _initAuthenticators(plugin) {
     const errorPrefix = `[${plugin.manifest.name}]:`;
 
     // @todo the _.isNil test need to be removed as soon as the
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)